frontend/src/main/java/cz/cuni/mff/xrg/odcs/frontend/gui/views/PipelineEdit.java

```diff
@@ -1178,6 +1188,9 @@ protected boolean savePipeline(final String successAction) {
         if (!validate()) {
             return false;
         }
+        if (!pipelineCanvas.validateGraph()) {
```

Comment:
well, so whole requirements on 'lets do not save blank pipeline' can be dropped too?